src/mdx/expandVersionedUrl.js

```diff
@@ -19,6 +19,10 @@
 /** @type {Array<VersionMapping>} */
 const versionMappings = [
   // 👋 When cutting a new version, add a new mapping here!
+  {
+    docsVersion: "8.3",
+    optimizeVersion: "3.11.0",
+  },
```

Comment:
oops! I forgot this, and it caused a _bunch_ of [broken links in the build]()

For future reference, if the build says this: 

```
- On source page path = /optimize/self-managed/optimize-deployment/version-policy/:
   -> linking to $docs$/reference/release-policy/ (resolved as: /optimize/self-managed/optimize-deployment/version-policy/$docs$/reference/release-policy/)
``` 

...it means the version it's building isn't defined in this array, so it doesn't know what to resolve `$docs$` or `$optimize$` to. (Note the `$docs$` in the broken link)